src/hotspot/share/services/management.cpp

```diff
@@ -1707,7 +1705,7 @@ JVM_ENTRY(jint, jmm_GetInternalThreadTimes(JNIEnv *env,
     MutexLocker ml(THREAD, Threads_lock);
     Threads::threads_do(&ttc);
   }
-  ttc.do_unlocked();
+  ttc.do_unlocked(THREAD);
```

Comment:
At first I thought this should use CHECK_0 not THREAD so it will return zero when an exception occurs. But this goes straight back to Java code so the exception will propagate and the return value can't be examined.